pairing/bn256/point.go

```diff
@@ -62,7 +62,34 @@ func (p *pointG1) Clone() kyber.Point {
 }
 
 func (p *pointG1) EmbedLen() int {
-	panic("bn256.G1: unsupported operation")
+	return (255 - 8 - 8) / 8
```

Comment:
Can you add a comment explaining the length ?

Reply:
Added what I thought was a reasonable comment.